src/network/networkd-manager.c

```diff
@@ -728,9 +728,6 @@ Manager* manager_free(Manager *m) {
         free(m->dynamic_timezone);
         free(m->dynamic_hostname);
 
-        safe_close(m->ethtool_fd);
-        safe_close(m->persistent_storage_fd);
```

Comment:
Issues like https://github.com/systemd/systemd/issues/32252 are caught with Valgrind. mkosi can't catch that.